docs/modules/ROOT/pages/ios_faq.adoc

```diff
@@ -0,0 +1,49 @@
+= iOS Frequently Asked Questions (FAQ)
+:hardbreaks:
+:oauth2-app-url: https://marketplace.owncloud.com/apps/oauth2
+:toc: right
+
+Here you can find some of the most frequently asked questions about the ownCloud iOS app.
+
+== Usage
+
+=== Compatibility with ownCloud server older than version 10.0
+
+Previous ownCloud iOS app with support for ownCloud server < 10.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
+
+=== Compatibility with iOS versions older than iOS 12.0
+
+Previous ownCloud iOS app with support for iOS < 12.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
+
+=== Login with TOTP and other 2FA
+
+ownCloud server must have {oauth2-app-url}[the OAuth2 app] installed, configured, and enabled to use Two-Factor Authentication.
+Please contact your ownCloud administrator for more details.
+
+== Feature Requests
```

Comment:
An empty headline is sensless and looks bad. Either remove or fill with content.